quickstarts/k6-prometheus/config.yml

```diff
@@ -0,0 +1,49 @@
+id: b9ef4c73-0875-43dd-b9d5-4c3bb74632ea
```

Comment:
My understanding is that new quickstarts should not have an ID field in the config? https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#fields. It looks like this is causing validation to fail because it's been copied over from the 'k6' quickstart config

Reply:
Thanks @sarahkitten, good catch, I missed that. id field is removed now.